spec/module.dd

```diff
@@ -634,6 +633,27 @@ $(GNAME MixinDeclaration):
         $(GLINK DeclDefs), and is compiled as such.
         )
 
+$(H2 $(LEGACY_LNAME2 AllocatorRelic, allocator-relic, Allocator Relic))
+
+$(GRAMMAR
+$(GNAME AllocatorRelic):
+    $(D new) $(D $(LPAREN)) $(D $(RPAREN)) $(D ;)
+)
+
+    $(P This is a relic of the old D1-style class and struct custom $(D new) operator.
+        Its sole purpose is to be prefixed with $(D @disable) so that $(D new) operator
+        is not permited for the aggregate that contains the allocator relic, for example
```

Comment:
permitted

Reply:
"NewClassDisabler" does not work. `@disable` is checked during semantics checks, it is still grammatically correct to have `new();` without the attribute.

Reply:
let's go for `NewDeclaration` then.